apps/src/studio/Item.js

```diff
@@ -315,10 +340,14 @@ Item.prototype.removeElement = function() {
 };
 
 /**
- * Stop our animations
+ * Retrieve animation frame duration (frames per tick)
  */
-Item.prototype.stopAnimations = function() {
-  this.animation_.stopAnimator();
+Item.prototype.getAnimationFrameDuration = function () {
+  if (this.dir === Direction.NONE) {
+    return this.normalFrameDuration;
```

Comment:
Interesting... so `speed` is something like a scale factor for both movement speed and movement animation rate?  I was under the impression that it was just an animation rate control.  Renaming to `movementSpeed` might be helpful, and/or a comment to that effect.

I thought it was weird to have frame duration depend on direction here, but I guess this is the Item class (game-level logic) not the animation class, so it's not that strange?